game/scripts/npc/items/item_trumps_fists_2.txt

```diff
@@ -112,8 +111,8 @@
             "12"
             {
                 "var_type"              "FIELD_INTEGER"
-                "heal_prevent_duration"  "5 5"
+                "heal_prevent_duration"  "5"
             }
 		}
-	}
-}
+	}
+}

```

Comment:
You have the wrong line endings here:
* You have: `<CRLR>`(Windows).
* You should have `<LF>`(Unix)